src/ServiceManagerBase.sol

```diff
@@ -168,6 +176,15 @@ abstract contract ServiceManagerBase is ServiceManagerBaseStorage {
         _setRewardsInitiator(newRewardsInitiator);
     }
 
+    /**
+     * @notice Sets the slasher address
+     * @param newSlasher The new slasher address
+     * @dev only callable by the owner
+     */
+    function setSlasher(address newSlasher) external onlyOwner {
```

Comment:
Do we plan on changing this because this is a bit concerning how the slasher can just be updated without any significant delay or additional security measures. Not sure if we want to have something in place by default or leave it up to AVSs because there are trust assumptions that operators have done their due diligence on AVSs they are securing.

Reply:
Initial implementation added [fc2a1ba](https://github.com/Layr-Labs/eigenlayer-middleware/pull/310/commits/fc2a1bac700dbdfa54fe484cec756a0cc26495bc)